cScripts/functions/modules/fn_zenModule_createSupplyCrate.sqf

```diff
@@ -14,31 +14,39 @@
  */
 
 params ["_modulePos", "_objectPos"];
+// The crate type that will be passed to the supply crate spawner.
+private _crateTypeList = [
+    'crate_resupply_general',
+    'crate_medicalAtlas', 
+    'crate_medicalInfantry',
+    'crate_stinger'
+];
 
+// Display text for the crate list selection. This is what the zeus sees.
+private _displayTextList = 
 [
-    "7th Cavalry Supply Crate", 
+    ['Infantry Resupply', "Contains weapons, ammo, and supplies for one infantry platoon"],
+    ['Atlas Team', "Contains enough supplies to sustain two Atlas teams as well supplementing one infantry platoon."],
+    ['Infantry Medical', "Contains enough medical supplies for one infantry platoon."],
+    ['Stinger MANPAD', "Contains 1 launcher and 2 missiles."]
+];
+[
+    "7th Cavalry Supply Crates", 
     [
-        ["SLIDER:PERCENT", ["Supply size", "Regulate the total amount of supplies in the crate"], [0, 1, 1], false]
+        ["LIST", ["Crate Type", "Select the type of unit you are supplying with this crate"],
+            [
+                _crateTypeList,
+                _displayTextList,
+                0,
+                4
+            ], false]
     ], 
     {
         params ["_arg", "_pos"];
-        _arg params ["_size"];
+        _arg params ["_crateType"];
         _pos params ["_modulePos"];
-
-        private _crate = "Box_NATO_Equip_F" createVehicle _modulePos;
         
-        [_crate, _size] remoteExec [QFUNC(doSupplyCrate), 0, true];
-
-
-        // Change ace characteristics of crate
-        [_crate, 1] call ace_cargo_fnc_setSize;
-        [_crate, true] call ace_dragging_fnc_setDraggable;
-        [_crate, true] call ace_dragging_fnc_setCarryable;
-
-        // Add object to Zeus
-        [{
-            _this call ace_zeus_fnc_addObjectToCurator;
-        }, _crate] call CBA_fnc_execNextFrame;
+        [_modulePos, _crateType] remoteExec [QFUNC(doSupplyCrate), 2, false];
```

Comment:
Think we can do a cba server event here later. but i think this is fine for now.

Reply:
We can merge this branch then keep it open for further edits. I want to merge it so we can update all the logistics macros